src/java/fr/paris/lutece/plugins/forms/web/FormXPage.java

```diff
@@ -143,7 +139,7 @@ public class FormXPage extends MVCApplication
     private static final String MARK_FORM_LIST = "form_list";
     private static final String MARK_DISPLAY_CAPTCHA = "display_captcha";
     private static final String MARK_CAPTCHA = "captcha";
-
+    private static final String AUTH_PROVIDER = "auth_provider";
```

Comment:
the auth_provider parameter is external to the forms plugin, you should not use it in  plugin forms code

Reply:
The goal of this PR is to simplify the way we detecte if we just logged in as an FO user, because the way it was done was quite complexe.
By using the the referer of the request, we know the url, and we can check the presence of key work.

So I can make some proposal : 
- We could put this constant (auth_provider) in src/java/fr/paris/lutece/plugins/forms/service/entrytype/EntryTypeMyLuteceUser.java , since it's already tied up to mylutece.
- We can check if the url contains "action=login" or just "login"
- We can deduce that it could come from a login if the url doesn't contains "forms" and "admin".

Let me know what you propose

Reply:
you cannot rely on the **_auth_provider_** attribute to know if a user has just connected because this attribute cannot always be retrieved from the request (a user can authenticate well before accessing the form)
can you explain why you want to do know if the user came to the login page ? 
Wouldn't it be enough to test if the _formResponseManager object is empty?

Reply:
"Wouldn't it be enough to test if the _formResponseManager object is empty?"
 In the form's configuration if you check "Enable response backup" without "Authentication required", you don't go to the login page first, but directly to the form. When you have loaded page of the form _formResponseManager is not null, but you supposed to be able to loggin and get you backup responses whenever. 

Detecting that you are coming from a login page thought the referer of the request, is much more simple than deducing it.

I have updaded this PR : implementing the login page parameters in forms.properties ( forms.frontOffice.loginPage.parameters=page=mylutece&action=login&auth_provider=mylutece-database ).
So the plugin user can change it, if the auth providers is not mylutece.